src/Adapter/Product/Combination/Update/CombinationStockUpdater.php

```diff
@@ -58,19 +58,27 @@ class CombinationStockUpdater
      */
     private $stockManager;
 
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
     /**
      * @param StockAvailableRepository $stockAvailableRepository
      * @param CombinationRepository $combinationRepository
      * @param StockManager $stockManager
+     * @param ConfigurationInterface $configuration
      */
     public function __construct(
         StockAvailableRepository $stockAvailableRepository,
         CombinationRepository $combinationRepository,
-        StockManager $stockManager
+        StockManager $stockManager,
+        ConfigurationInterface $configuration
```

Comment:
I guess no BC break because class not released in 1.7.8.0 ?

Reply:
It was in 178 apparently, so its a BC :bow: . Updated description